resources/views/layout/metadata.blade.php

```diff
@@ -28,6 +28,20 @@
 <meta name="keywords" content="osu, peppy, ouendan, elite, beat, agents, ds, windows, game, taiko, tatsujin, simulator, sim, xna, ddr, beatmania, osu!, osume">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+@if ($hasOpengraph)
+    <meta property="og:site_name" content="{{ 'osu! » '.$opghSitename ?? 'osu!' }}">
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="{{ Request::url() }}">
```

Comment:
this should be canonical url (e.g. `https://osu.ppy.sh/users/1` even when the user accesses `https://osu.ppy.sh/users/1?random=parameter`).